system-tests/.gitignore

```diff
@@ -1,2 +1,7 @@
+# Hardhat
+/build/
 /cache/
+/export/
 /external/
+/typechain/
+/export.json
```

Comment:
There is a convention to have an empty line at the end of all files.

Reply:
Sure [f2e884f](https://github.com/keep-network/tbtc-v2/pull/421/commits/f2e884f2d18612de52065cf053b4282550cc9ac8)